.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -15,6 +15,10 @@ steps:
       filePath: yarn-ci.sh
     displayName: yarn (install packages)
 
+  - script: |
+      yarn playwright install --with-deps
```

Comment:
I'm wondering what side-effects this might have for our VR testing solutions. can we reach out to folks @TristanWatanabe and  test it out locally ?

Reply:
Best way to test would be to apply the changes in this PR and trigger VR testing for v0, v8 and v9 (via a PR) and see if any new discrepancies are reported

Reply:
Yep @radium-v , can you just make a random change to a `src/` file in the `@fluentui/react-components` package to trigger the VR tests so we can see the diff?

Reply:
@TristanWatanabe I made a small change which triggered the VR tests. I'm new to the process but the vast majority of the diffs look like noise.

Reply:
yeah, that's a good thing as we can proceed